Apps/frontend/src/const.js

```diff
@@ -52,3 +52,10 @@ export const BACKEND_BASE_URL = import.meta.env.VITE_BACKEND_BASE_URL;
 export const BACKEND_API_URL = `${BACKEND_BASE_URL}`;
 export const REST_ENDPOINT_CONFIG = `${BACKEND_API_URL}/config`;
 /* END REST API CONFIG */
+
+/* DISPLAY SPEED */
+const EXPECTED_UPDATES_PER_SECOND = 10_000;
```

Comment:
I'm not a fan of this, but due to the short time until the end of the project, I think it's reasonable that we assume a fixed frequency.

Reply:
As a next step, the backend can measure the number of packages received from the generator and notify the front end about this. I am pretty sure a real digital oscilloscope would also need to know about the sampling rate in order to plot the signal with the set time/div value.